### PR TITLE
[MRG] add `sum_hashes` to `sourmash sig describe` output.

### LINKED ISSUE #1833: lca summarize abundance with outfile
Hi,

thanks for this software. I'm trying sourmash as an alternative for abundance estimation. When I run the `sourmash lca summarize` command, it works fine and reports in stdout what seems to be abundance values (at least a percentage). However, when I define an output file with the same query and database , this column does not appear and I only have count. 

Sorry if I missed something here.
sourmash 4.2.4 

Regards,
JSG

### PR DESCRIPTION
**Pull request by ctb**
This PR -
* adds `sum hashes:` to human readable output, and `sum_hashes` to CSV output, for `sig dsecribe`
* adds `--include-db-pattern` and `--exclude-db-pattern` to `sig describe`
* fixes a bug in the CSV output of `sig describe` where `signature_file` was always empty;
* removes a confusing and useless test file, and updates tests to use a less confusing file;

Fixes https://github.com/sourmash-bio/sourmash/issues/1833 by making `sum_hashes` visible (the last remaining part of that issue).

### CODE REVIEW

**Comment by ctb** [comment]
ready for review & merge @sourmash-bio/devs

**Review by bluegenes — APPROVED** [review]
Lgtm!

Just so I'm clear - sum hashes is the sum of the abundances for all unique hashes?

**Comment by ctb** [comment]
> Lgtm!
> 
> Just so I'm clear - sum hashes is the sum of the abundances for all unique hashes?

Yep! Added some doc text in 065b05bc

File: doc/command-line.md [hunk 1]
```diff
@@ -989,20 +989,25 @@ Display signature details.
...
-k=31 molecule=DNA num=0 scaled=1000 seed=42 track_abundance=0
+k=31 molecule=DNA num=0 scaled=1000 seed=42 track_abundance=1
 size: 5177
+sum hashes: 5292
 signature license: CC0
 ```
 
+Here, the `size` is the number of distinct hashes in the sketch, and
+`sum_hashes` is the total number of hashes in the sketch, with abundances.
+When `track_abundance` is 0, `size` is always the same as `sum_hashes.
```
Comment (bluegenes) [inline doc/command-line.md]: is this missing a closing "`"?

**Pull request merged**